src/query/ast/src/ast/statements/statement.rs

```diff
@@ -283,6 +284,20 @@ impl Statement {
                 }
                 format!("{}", Statement::CreateStage(stage_clone))
             }
+            Statement::AttachTable(attach) => {
+                let mut attach_clone = attach.clone();
+                attach_clone.uri_location.connection = attach_clone.uri_location.connection.mask();
+                format!("{}", Statement::AttachTable(attach_clone))
+            }
+            Statement::CreateConnection(stmt) => {
+                let mut stmt_clone = stmt.clone();
+                stmt_clone.storage_params = stmt_clone
+                    .storage_params
```

Comment:
Impl a debug with mask for storage_params?

Reply:
done, impl in Display, leave normal Debug to developer for debugging